examples/platform/silabs/efr32/init_efrPlatform.cpp

```diff
@@ -65,7 +66,16 @@ void initAntenna(void);
 void init_efrPlatform(void)
 {
     sl_system_init();
-    sl_mbedtls_init();
+
+#if CHIP_ENABLE_OPENTHREAD
+#ifdef MGM24
+    sl_openthread_init();
```

Comment:
If I recall right, this call would cause an assert using our thread cert lib, the underlying otSysInit wasn't implemented the same way in the gsdk with the lib than with the thread source code. Can you just give it a quick test?

Reply:
I would say that if it cause an Assert, then something is wrong with the Cert Lib for the modules. And I cannot test it since there is no cert lib for the module.